macosx/Base.lproj/PrefsWindow.xib

```diff
@@ -89,7 +89,6 @@
                                 <font key="font" metaFont="system"/>
                             </buttonCell>
                             <connections>
-                                <action selector="setAutoUpdateToBeta:" target="-2" id="1900"/>
```

Comment:
Has this been sanity checked to still work with the binding?

Reply:
That's how the other checkboxes work... if that was not working, many more things would be broken.

Reply:
Is this topic resolved?

Reply:
@nevack @livings124 @Coeur Is this topic resolved?

Reply:
I did the sanity check:
```
(lldb) po [NSUserDefaults.standardUserDefaults boolForKey:@"AutoUpdateBeta"]
YES
(lldb) po [NSUserDefaults.standardUserDefaults boolForKey:@"AutoUpdateBeta"]
NO
```
So that topic is resolved, the checkbox is working fine like any other checkbox in the settings.